core/designsystem/src/main/kotlin/org/mifospay/core/designsystem/component/AlertDialog.kt

```diff
@@ -10,15 +10,15 @@ import androidx.compose.ui.window.Dialog
 
 @Composable
 fun MifosDialogBox(
-    showDialogState: MutableState<Boolean>,
+    showDialogState: Boolean,
```

Comment:
You have made changes in core designsystem component, make sure its working properly for other screens as well wherever it is used.